expr/func_aliasbynode.go

```diff
@@ -33,6 +33,9 @@ func (s *FuncAliasByNode) Exec(cache map[Req][]models.Series) ([]models.Series,
 	}
 	for i, serie := range series {
 		metric := extractMetric(serie.Target)
+		if len(metric) == 0 {
```

Comment:
in which case do we expect this (meaning specifically: get a "" back and it being valid) to happen?

Reply:
for something like `aliasByNode(sumSeries(seriesByTag('host=~h.*9','name=~cpu.*')),0)` there is no metric name / path expression to extract. In this case, the function expression is perfectly valid, but we can't pick out the name. Now, it's possible that there is one matching name (e.g. cpu.percent.idle) in which case we can pick it out of the `name` tag. This is a fairly contrived example, but it basically matches the behavior of [this recent graphite change](https://github.com/graphite-project/graphite-web/pull/2124/files)

Except now that I'm looking, I see that I forgot to split on `;`

Reply:
makes sense, please add that as a comment in the code.